src/layouts/product-integration-layout/components/header/index.tsx

```diff
@@ -49,41 +59,83 @@ export default function Header({
 		.map((f: Flag) => f.slug)
 		.includes('builtin')
 
+	const tags = GetIntegrationTags(integration, true)
+
 	return (
-		<Card className={classNames(s.header, className)}>
-			<div className={s.upperCard}>
-				<div className={s.left}>
-					<h1>{integration.name}</h1>
-					<span>@{integration.organization.slug}</span>
-				</div>
-				<div className={s.right}>
-					{showVersions ? (
-						<DropdownDisclosure
-							className={s.versionDropdown}
-							color="secondary"
-							text={versionString(activeRelease.version, integration.versions)}
+		<Card elevation="base" className={s.headerWrap}>
+			<IconTileLogo
+				className={s.iconTile}
+				size="large"
+				productSlug={productSlug !== 'sentinel' ? productSlug : null}
```

Comment:
This ternary is to make typescript happy, since we don't support sentinel yet but technically its a valid product slug 🙃